rpc/client_test.go

```diff
@@ -0,0 +1,564 @@
+package rpc_test
+
+import (
+	"github.com/joeqian10/neo-gogogo/rpc"
+	"log"
+	"testing"
+)
+
+var LocalEndPoint = "http://localhost:50003" // if you want to test, you need to change this endpoint to yours
```

Comment:
It will be more convenient for other developers to run test if we use `TestNetEndPoint`.